src/test/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringcreateanorderapi/model/fms/MonitoringOrderTest.kt

```diff
@@ -246,10 +259,107 @@ class MonitoringOrderTest : OrderTestBase() {
     assertThat(fmsMonitoringOrder.pduResponsible).isEqualTo(mappedValue)
   }
 
+  @ParameterizedTest(name = "it should map civilCountyCourt - {0} -> {1}")
+  @ArgumentsSource(CivilAndCountyCourtArgumentsProvider::class)
+  fun `It should correctly map saved civilCountyCourt values to Serco`(savedValue: String, mappedValue: String) {
+    val order = createOrder(
+      dataDictionaryVersion = DataDictionaryVersion.DDV5,
+      deviceWearer = createDeviceWearer(),
+      interestedParties = createInterestedParty(
+        responsibleOrganisation = "PROBATION",
+        notifyingOrganisationName = savedValue,
+      ),
+    )
+    val fmsMonitoringOrder = MonitoringOrder.fromOrder(order, null)
+
+    assertThat(fmsMonitoringOrder.noName).isEqualTo(mappedValue)
+  }
+
+  @ParameterizedTest(name = "it should map crownCourt - {0} -> {1}")
+  @ArgumentsSource(CrownCourtArgumentsProvider::class)
+  fun `It should correctly map saved crownCourt values to Serco`(savedValue: String, mappedValue: String) {
+    val order = createOrder(
+      dataDictionaryVersion = DataDictionaryVersion.DDV5,
+      deviceWearer = createDeviceWearer(),
+      interestedParties = createInterestedParty(
+        responsibleOrganisation = "PROBATION",
+        notifyingOrganisationName = savedValue,
+      ),
+    )
+    val fmsMonitoringOrder = MonitoringOrder.fromOrder(order, null)
```

Comment:
A lot of duplication here, can we warp this in one method, and have other notifying org test calling this method?